### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review

### CODE REVIEW

File: Modal.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    border-radius: 3px;
+    color: var(--component-container-color);
+  }
+  .modal-container {
+    align-items: center;
+    background-color: var(--component-container-color);
+    color: var(--component-black-color);
+    border-radius: 0.3rem;
+    display: flex;
+    gap: 3rem;
```
Comment (gadgetfather) [inline Modal.css]: You can remove this gap to make the button center.

File: common.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans&family=Poppins&family=Roboto&family=Roboto+Condensed&display=swap');
+*
+body 
+{
+    font-family: "Roboto", sans-serif;
+    font-size: 1rem;
```
Comment (gadgetfather) [inline common.css]: By default font-size is 16px. no need to redeclare.

**Review by gadgetfather — CHANGES_REQUESTED** [review]

File: Modal.css (hunk 1 above)
Comment (b0llu) [inline Modal.css]: due to the gap the justify center is a little left. 
image

**Comment by ashishkatiyar2404** [comment]
your modal is awesome💯.
code also looks good to merge.

File: Modal.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    opacity: 1;
+  }
+  .modal-content {
+    position: relative;
+    background-color: var(--component-white-color);
+    width: 90%;
+    max-width: 24rem;
+    padding: 2rem 1rem;
+    border-radius: 4px;
+    /* font-family: Arial, Helvetica, sans-serif; */
```
Comment (b0llu) [inline Modal.css]: remove commented code if not going to use it in future

**Review by b0llu — APPROVED** [review]

File: Modal.css (hunk 1 above)
Comment (ShrutiLakhani) [reply Modal.css]: Done. Thanks for pointing it out

File: common.css (hunk 2 above)
Comment (ShrutiLakhani) [reply common.css]: Done.

File: Modal.css (hunk 1 above)
Comment (ShrutiLakhani) [reply Modal.css]: Done. Thanks for pointing it out.

File: Modal.css (hunk 3 above)
Comment (ShrutiLakhani) [reply Modal.css]: Done

**Review by gadgetfather — APPROVED** [review]

**Pull request merged**